src/_includes/navbar.njk

```diff
@@ -12,8 +12,9 @@
   </div>
   <div id="menu"
     class="w-full sm:w-auto self-end sm:self-center hidden sm:flex flex-col sm:flex-row items-center h-full py-1 pb-4 sm:py-0 sm:pb-0">
-    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center no-underline py-2 mx-2" href="{{ env.baseUrl }}coc">Code Of Conduct</a>
-    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center no-underline py-2 mx-2" href="https://in.pycon.org/blog/">Blog</a>
+    
+    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center w-full no-underline py-2 mr-6" href="{{ env.baseUrl }}cfp">CFP</a>
```

Comment:
I think we should have "Call For Proposal" since we are using "Code Of Conduct" instead of CoC :)